easybuild/easyconfigs/m/MIGRATE/MIGRATE-3.4.4-goolf-1.4.10.eb

```diff
@@ -0,0 +1,46 @@
+# 
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+# New Zealand eScience Infrastructure (NeSI)
+
+name = 'MIGRATE'
+version = "3.4.4" 
+easyblock = 'ConfigureMake'
+
+homepage = 'http://popgen.sc.fsu.edu/Migrate/Migrate-n.html'
+description = """Migrate estimates effective population sizes and past migration rates between n population assuming a migration matrix model with asymmetric migration rates and different subpopulation sizes. Migrate uses maximum likelihood or Bayesian inference to jointly estimate all parameters."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'} 
+toolchainopts = {'opt': True, 'optarch': True, 'usempi': True, 'unroll': True, 'static': False}
+
+# For sources line to work correctly with --try-software-version, you MUST employ %s OR use a construct like SOURCE_TAR_GZ
+oldversions="oldversions/%s.x/%s/" % (".".join(version.split('.')[:1]), ".".join(version.split('.')[:2])) 
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+source_urls = ['http://popgen.sc.fsu.edu/newversions/', 'http://popgen.sc.fsu.edu/currentversions/', 'http://popgen.sc.fsu.edu/%s' % oldversions ]
+
+patches = []
+
+dependencies = [('zlib', '1.2.5')]
+
+preconfigopts = 'cd ./src && sed -i "s/ \-fast / /g" configure && ' 
```

Comment:
use `start_dir = 'src'`

I'm not a big fan of using `sed`, but we don't have a better alternative yet (other than creating a patch)